src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (stylesString) => {
+  const styles = {};
+
+  const declarations = stylesString.split(';');
```

Comment:
Consider using a more descriptive variable name than `declarations`. Since this variable represents an array of individual CSS declarations, a name like `cssDeclarations` would be more informative and improve code readability.